python/cuda_cccl/pyproject.toml

```diff
@@ -16,27 +16,49 @@ classifiers = [
   "License :: OSI Approved :: Apache Software License",
 ]
 requires-python = ">=3.9"
+
 dependencies = [
   "numba>=0.60.0",
```

Comment:
Q: Do we need to include numba explicitly, or can we have it through numba-cuda?

Reply:
If the code uses `numba` directly (not `numba-cuda`) for extensions - then yes